packages/components/psammead-bulletin/src/index.jsx

```diff
@@ -0,0 +1,277 @@
+import { getSansRegular, getSansBold } from '@bbc/psammead-styles/font-styles';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import { getPica, getLongPrimer } from '@bbc/gel-foundations/typography';
+import { mediaIcons } from '@bbc/psammead-assets/svgs';
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
+
+const BulletinWrapper = styled.div`
+  background-color: ${C_WHITE};
+  padding: ${GEL_SPACING};
+  position: relative;
+  text-decoration: none;
+
+  @media (min-width: ${GEL_GROUP_CD_MIN_WIDTH}rem) {
+    display: none;
+  }
+`;
+
+const IconWrapper = styled.span`
+  > svg {
+    color: ${C_WHITE};
+    fill: currentColor;
+    margin-right: ${GEL_SPACING};
+    height: ${GEL_SPACING_DBL};
+  }
+`;
+
+const PlayCTA = styled.div.attrs({ 'aria-hidden': true })`
+  background-color: ${({ isLive }) => (isLive ? C_POSTBOX : C_EBON)};
+  color: ${C_WHITE};
+  width: 100%;
+  display: block;
+  text-align: center;
+  padding: ${GEL_SPACING};
+  vertical-align: middle;
+`;
+
+const StyledSummary = styled.p`
+  ${({ script }) => script && getLongPrimer(script)};
+  ${({ service }) => getSansRegular(service)}
+  color: ${C_SHADOW};
+  margin: 0; /* Reset */
+  padding: ${GEL_SPACING} 0 ${GEL_SPACING_DBL};
+`;
+
+const StyledHeading = styled.h3`
+  ${({ script }) => script && getPica(script)}
+  ${({ service }) => getSansBold(service)}
+  color: ${C_EBON};
+  margin: 0; /* Reset */
+`;
+
+const HeadingLink = styled.a`
+  position: static;
+  color: ${C_EBON};
+  text-decoration: none;
+
+  &:before {
+    bottom: 0;
+    content: '';
+    left: 0;
+    overflow: hidden;
+    position: absolute;
+    right: 0;
+    top: 0;
+    white-space: nowrap;
+    z-index: 1;
+  }
+
+  &:visited {
+    color: ${C_METAL};
+  }
+
+  :hover,
+  :focus {
+    text-decoration: underline;
+  }
+`;
+
+const StyledLiveIndicator = styled.span`
+  color: ${C_POSTBOX};
+  text-transform: uppercase;
+  text-decoration: none;
+  :hover {
+    text-decoration: none;
+  }
+`;
+
+const BulletinBodyWrapper = styled.div`
+  padding-top: ${GEL_SPACING};
+`;
+
+const BulletinImageWrapper = styled.div`
+  padding-bottom: ${GEL_SPACING};
+`;
+
+const StyledBulletinImage = styled.img`
+  display: block;
+  width: 100%;
+  margin: 0;
+  padding: 0;
+`;
+
+const BulletinBody = ({
+  title,
+  summary,
+  type,
+  liveText,
+  script,
+  service,
+  link,
+  CTAText,
+  hiddenTitleText,
+  dir,
+}) => (
+  <BulletinBodyWrapper dir={dir}>
+    <StyledHeading script={script} service={service}>
+      <span>
+        <VisuallyHiddenText>{hiddenTitleText}</VisuallyHiddenText>
+        {liveText && <LiveIndicator text={liveText} />}
+        <HeadingLink href={link} dir={dir}>
+          {title}
+        </HeadingLink>
+      </span>
+    </StyledHeading>
+    <StyledSummary script={script} service={service}>
+      {summary}
+    </StyledSummary>
+    <PlayCTA type={type} isLive={liveText && true} dir={dir}>
```

Comment:
What is the reason for adding `&& true` here?

Reply:
There isn't one. Not even sure why I put it in in the first place :D